Uplift/Supporting/Constants.swift

```diff
@@ -77,7 +77,7 @@ struct Identifiers {
             return "Mindfulness"
         }
     }
-    static let hasSeenOnboarding = "hasSeenOnboarding2.0"
+    static let hasSeenOnboarding = "hasSeenOnboarding3.0"
```

Comment:
changing this means any one who's already been through the onboarding slides but not the select favorite gyms page will go through the entire flow again, right?